tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -105,6 +105,7 @@ namespace Datadog.Trace.Configuration
         public bool KafkaCreateConsumerScopeEnabled { get; }
         public bool LogsInjectionEnabled { get; }
         public int MaxTracesSubmittedPerSecond { get; }
+        public int PropagationHeaderMaxLength { get; }
```

Comment:
These don't seem like they need to be public to me? 🤔

Reply:
Resolving as [dupe](https://github.com/DataDog/dd-trace-dotnet/pull/2897#discussion_r911033747).